cap_states.proto

```diff
@@ -21,6 +21,7 @@ message CapState {
   optional TimeWithZone start_date = 10;
   optional TimeWithZone end_date = 11;
   optional int64 current = 12;
-  optional int64 limit = 13;
+  optional int64 deprecated_limit = 13;
```

Comment:
Can you give this a better name? Right now it looks like a CapState has a "limit" and another value "deprecated limit". Maybe `limit_DEPRECATED`? Or just use `reserved 13;` (with comment)?